lib/dfid-transition/transform/unescape_html.rb

```diff
@@ -0,0 +1,12 @@
+require 'cgi'
+
+module DfidTransition
+  module Transform
+    module UnescapeHtml
```

Comment:
I was trying to name this module in such a way that it clearly had a very small and concise task that it would do well and wouldn't be a place to dump other stuff into, like `strip`.

I do like your method naming suggestion, I should have gone back and refactored under green tests.